packages/logger/README.md

```diff
@@ -0,0 +1,27 @@
+# Appium Logger
+
+The logger util that Appium uses.
+
+# Installation
+
+```console
+npm install @appium/logger --save
+```
+
+# Basic Usage
+
+```js
+import log from '@appium/logger';
+
+// additional stuff ---------------------------+
+// message ----------+                         |
+// prefix ----+      |                         |
+// level -+   |      |                         |
+//        v   v      v                         v
+    log.info('fyi', 'I have a kitty cat: %j', myKittyCat);
+```
+
+# History
+
+This module is forked from [npmlog](https://github.com/npm/npmlog) as [appium-logger](https://github.com/appium/appium-logger).
```

Comment:
I will delete the appium-logger repository and only keep this one, so links must point to this package